__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const { app } = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+beforeEach(() => seed(testData));
+
+describe.only("API Testing", () => {
+    describe("Ticket 3: GET /api/topics", () => {
+      test("returns an array of topics", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
```

Comment:
remember to check that the length of the array is what you expect, as a forEach without a length check could yield a false positive